main.js

```diff
@@ -0,0 +1,57 @@
+var weather = [];
+var apiKey = 'bbc14ddfa56edcf6519c2efeb3c1ba71';
+
+$('.search').on('click', function () {
+  var city = $('#search-query').val();
+
+  //console.log('click');
+  fetch(city);
+
+});
+var addWeather = function (data) {
+  weather = [];
+   
+  for (var i = 0; i < data.items.length; i++) {
+    var currentWeatherData = data.items[i];
+   
+    var currentWeather = {
```

Comment:
I'm curious about why you're using all of the weather data instead of of simply pieces or averages. Some comments might be nice here.